src/containers/CollectionPage.js

```diff
@@ -37,10 +39,17 @@ class CollectionPage extends React.Component {
   };
 
   render() {
-    const { collections, collection, publicFolder, page, entries } = this.props;
+    const { collections, collection, publicFolder, page, entries, isFetching } = this.props;
     if (collections == null) {
       return <h1>No collections defined in your config.yml</h1>;
     }
+
+    const noEntriesContent = isFetching
```

Comment:
Didn't notice the whole thing was in a ternary, so yeah, that's best. We can clean it up further by also storing the `EntryListing` render in a `const` and one-lining the ternary.